composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
```

Comment:
require-dev should be below require, just cos :-)

Reply:
That 1000000000% does not matter, but I'll make that change to humour you since I have to touch this file anyway. :p